cmds/boot/systemboot/main.go

```diff
@@ -45,6 +45,8 @@ func isFlagPassed(name string) bool {
 }
 
 var defaultBootsequence = [][]string{
+	{"pxeboot"},
+	{"boot"},
 	{"fbnetboot", "-userclass", "linuxboot"},
 	{"localboot", "-grub"},
```

Comment:
@insomniacslk should we remove localboot from the hard-coded defaultBootsequence or keep it around? And do you have any preference about the boot order sequence here for Facebook's use case?